### DISCUSSION TITLE
Why vim-lsp for neovim when neovim has native LSP support?

### DISCUSSION THREAD

**Discussion by naggie** [prabirshrestha/vim-lsp#1467] [Q&A]
I'm sure there's a reason! I am not sure, after reading the README. Thanks!

**Comment by prabirshrestha** [comment]
vim-lsp existed before neovim supported it. It was one of the first one lsp plugins for the vim ecosystem. It is so old that the time I wrote vim-lsp, I had to convince folks on the internet why LSP is awesome.

**Comment by naggie** [comment]
I see! Thanks @prabirshrestha
